src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -209,8 +211,9 @@ def save_registration_data(
             submission=submission
         )
 
-        with get_documents_client() as documents_client, save_and_raise(
-            registration_data
+        with (
+            get_documents_client() as documents_client,
+            save_and_raise(registration_data),
```

Comment:
black had opinions :shrug:

Reply:
Interesting, are you running under Python 3.10?

Reply:
Yes, 3.10.13. Are you not?

Reply:
3.10.13 as well, sometimes the Python version used to run black can alter the formatting, but in this case I have no idea why :shrug: